site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -273,6 +290,9 @@ The id of the Key used to sign the SAML response. If you do not specify this pro
 [field]#application.samlv2Configuration.logoutURL# [type]#[String]# [optional]#Optional# [default]#Defaults to the system logout URL or `/`# [since]#Available since 1.6.0#::
 The URL that the browser is taken to after the user logs out of the SAML service provider. Often service providers need this URL in order to correctly hook up single-logout. **Note** that FusionAuth does not support the SAML single-logout profile because most service providers to not support it properly.
 
+[field]#application.samlv2Configuration.requireSignedRequests# [type]#[Boolean]# [optional]#Optional# [default]#Defaults to `false`# [since]#Available since 1.20.0#::
+Set this parameter equal to `true` to require the SAML v2 Service Provider to sign the request. When this value is `true` all requests missing a signature will be rejected and [field]#application.samlv2Configuration.defaultVerificationKeyId# is required.
```

Comment:
This second sentence needs to be re-worded. 

> When this value is `true` all requests missing a signature will be rejected and [field]#application.samlv2Configuration.defaultVerificationKeyId# is required.

These are two separate thoughts. Usually the contextual description of an optional field being required is on a new line, similar to how you did for the `defaultVerificationKeyId` above.